auth.go

```diff
@@ -389,26 +407,35 @@ func (x *Central) GetTokenFromIdentityPassword(identity, password string) (*Toke
 // The session key is typically sent to the client as a cookie.
 func (x *Central) Login(identity, password string) (sessionkey string, token *Token, e error) {
 	token = &Token{}
-	if token.Identity, e = x.authenticator.Authenticate(identity, password); e == nil {
-		x.Log.Infof("Login authentication success (%v)", identity)
-		var permit *Permit
-		if permit, e = x.permitDB.GetPermit(identity); e == nil {
-			if x.MaxActiveSessions != 0 {
-				if e = x.sessionDB.InvalidateSessionsForIdentity(identity); e != nil {
-					x.Log.Errorf("Invalidate sessions for identity (%v) failed when enforcing MaxActiveSessions (%v)", identity, e)
-					return "", nil, e
+	if e = x.authenticator.Authenticate(identity, password); e == nil {
```

Comment:
Let's rather rewrite this function in the style:
```go
if x, e = something(); e != nil {
   return
}
if y, e = something(); e != nil {
    return
}
```
so that the code reads linearly, with the success case at the end.
You may also want to get rid of the named return values, if that adds any clarity, but that's hard to judge without actually writing the code. The important thing is that if an error is returned, then the return value MUST be `("", nil, err)`. I find that easier to guarantee when one doesn't have named return values.